trs_cli/client.py

```diff
@@ -88,6 +94,81 @@ def __init__(
     #
     # Check DRS-cli repo for examples
 
+    def get_descriptor(
+        self,
+        type: str,
+        id: str,
+        version_id: str,
+        token: Optional[str] = None
+    ) -> Union[FileWrapper, Error]:
+        """Get the tool descriptor for the specified tool.
+
+        Arguments:
+            type: The output type of the descriptor. Plain types return
```

Comment:
Describe the behavior for passing TRS URIs, analogous to DRS URIs in DRS-cli, but with the difference that version IDs exists here as well. Basically:

* TRS URIs can be passed to `id` instead of a bare tool identifier
* If TRS URIs include the version information, passing a `version_id` is optional
* If a `version_id` is provided although the TRS URI already contains that information, the explicitly passed `version_id` will override the one extracted from the TRS URI